tests/test_utils.py

```diff
@@ -260,22 +260,30 @@ def test_calculate_alive_path(example_transaction_data, example_summary_data, fi
 
 
 def test_check_inputs():
-    freq, recency, T = np.array([0,1,2]), np.array([0, 1, 10]), np.array([5, 6, 15])
-    assert utils._check_inputs(freq, recency, T) is None
+    frequency = np.array([0, 1, 2])
```

Comment:
Renamed `freq` to `frequency` for consistency and added testing of `monetary_value`